### ISSUE TITLE
Fullscreen in mpv is locked after choosing split

### ISSUE THREAD

**Issue by mielouk** [swaywm/sway#4753]
* Sway Version: 1.2

* Debug Log:
https://8n1.org/16317/30f5

* Configuration File:
https://8n1.org/16318/2d8b

Finally, explain the steps you took in plain English to reproduce the problem below.

If [mpv](https://github.com/mpv-player/mpv) is in fullscreen and the split direction (h/v) is chosen, the fullscreen can not be un-done. Neither with double-click, nor F, nor the OSD. To get out you have to kill/close the program.

**Comment by emersion** [comment]
Can you try on master? I can't reproduce locally.

**Comment by mielouk** [comment]
`WAYLAND_DEBUG=1 mpv`
https://8n1.org/16321/0bf6

[This](https://github.com/swaywm/sway/wiki/Development-Setup) is equal to master? As the AUR provides a simple way this would be a nice solution to try this. Otherwise, do you have a list of commands to pull, compile and install 'master'?

**Comment by Vorsplummi** [comment]
One workaround is to use Sway's own fullscreen toggle. 

Firefox and Chromium do the same thing were they stay fullscreened when you toggle a split and open new window while playing video for example. They will honour F11 however.

![pseudofullscreen](https://user-images.githubusercontent.com/16557949/69624763-f197b900-104d-11ea-87b8-75bebe264ae7.png)

**Comment by mielouk** [comment]
Are you able to reprodue this problem?

Trying not to use the split shortcut during fullscreen is my workaround, which doesn't always work ;) I'm aware of those workarounds, but it involves more than one keystroke, like F or Esc. Not really feasible in my workflow.

**Comment by RPigott** [comment]
Hmm. I also can't reproduce on master.

@mielouk Yes those are instructions for compiling from master. The aur has a [sway-git](https://aur.archlinux.org/packages/sway-git/) pkgbuild that builds from master as well.

**Comment by Vorsplummi** [comment]
I can reproduce from master. I tried to make a short video to highlight the behaviour:

https://youtu.be/0K4d-GnXQ5w

I tried to use wshowkeys to show how MPV wouldn't accept fullscreen shortcut but it doesn't draw over fullscreen windows even if it's not truly fullscreen.

**Comment by mielouk** [comment]
Just installed sway-git, wlroots-git and swaybg-git from the AUR. The problem still persists. I finally did it because another bug appeared, which was solved by master. (DnD files from filemanager into mpv and mpv didn't respond any more to any input. This was not reproducible under X (Cinnamon). Solved by master.)
Though split screen while mpv is in full screen still locks it into full screen.

**Comment by edrex** [comment]
sameas #4616

**Comment by mielouk** [comment]
For me and my use case it is fixed. Close at your discretion.
